<ui version="4.0" >
 <class>prefs_identity</class>
 <widget class="QWidget" name="prefs_identity" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>375</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Default &amp;Language:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>E&amp;mail:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>kcfg_Email</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel2_2" >
     <property name="text" >
      <string>&amp;Name:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="kcfg_authorName" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Default Mailing L&amp;ist</string>
     </property>
     <property name="buddy" >
      <cstring>kcfg_DefaultMailingList</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="kcfg_DefaultMailingList" />
   </item>
   <item row="2" column="1" >
    <widget class="KLanguageButton" native="1" name="DefaultLangCode" />
   </item>
   <item row="5" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="kcfg_authorEmail" />
   </item>
   <item row="4" column="1" >
    <widget class="QLineEdit" name="kcfg_DefaultLangCode" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KLanguageButton</class>
   <extends>QWidget</extends>
   <header>klanguagebutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global" >klanguagebutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
